### PR TITLE
Add many descriptions to the `Window` class

### PR DESCRIPTION
**Pull request by SirQuartz**
Added several missing descriptions for various functions and properties. I don't fully understand what every function of this node does yet, but I think I'm on the right track with these additions after some testing. Plus it's better than nothing. I intend to improve this wherever possible.

If you've any feedback, recommendations, and/or insights I'd be glad to hear it.

### CODE REVIEW

**Comment by foxydevloper** [comment]
The descriptions of the properties should be changed to not start with "sets" or "returns", since properties can both be set and return. You can look through the docs to see how other other classes' properties' descriptions are written.
For example, instead of "Returns `true` if the window can't be resized.", you could write "If `true`, the window can't be resized." which is more consistent the rest of the documentation.

File: doc/classes/Window.xml [hunk 1]
```diff
@@ -333,16 +343,21 @@
 		<member name="min_size" type="Vector2i" setter="set_min_size" getter="get_min_size" default="Vector2i( 0, 0 )">
 		</member>
 		<member name="mode" type="int" setter="set_mode" getter="get_mode" enum="Window.Mode" default="0">
+			Sets the window's current mode (i.e windowed, fullscreen, minimized, etc).
+			[b]Note:[/b] Fullescreen mode is not exclusive fullscreen on Windows and Linux.
```
Comment (foxydevloper) [inline doc/classes/Window.xml:347]:
```suggestion
			[b]Note:[/b] Fullscreen mode is not exclusive fullscreen on Windows and Linux.
```

**Comment by SirQuartz** [comment]
> The descriptions of the properties should be changed to not start with "sets" or "returns", since properties can both be set and return. You can look through the docs to see how other classes' properties' descriptions are written.
> For example, instead of "Returns `true` if the window can't be resized.", you could write "If `true`, the window can't be resized." which is more consistent the rest of the documentation.

Thank you! I'll see to fixing that!.

**Comment by foxydevloper** [comment]
It seems you also changed the descriptions for the methods, those make sense to start with "Return". I was only talking about the classes' properties. Otherwise the change look good.

**Comment by SirQuartz** [comment]
> It seems you also changed the descriptions for the classes' methods, those made sense to start with "Returns". I only suggested changing the classes' properties' descriptions. Otherwise the change look good.

Thanks! I've updated the classes' methods to start "Returns" again!

**Comment by pycbouh** [comment]
Please squash your commits. You need to use a desktop Git client for that, you can't do it using GitHub on web.

https://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html

**Comment by SirQuartz** [comment]
> Please squash your commits. You need to use a desktop Git client for that, you can't do it using GitHub on web.
> 
> https://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html
> 
> You will have to do this after reviews as well, so it's a good time to practice now.

Ok, I think I've managed to do it! Thank you!

**Comment by Chaosus** [comment]
> Please squash your commits.

That deadly advice :D

**Comment by SirQuartz** [comment]
> > Please squash your commits.
> 
> That deadly advice :D

Yeah, I'll be honest, I'm not very familiar with Git, and I'm still learning a lot about it, so things got a _little_ messy. Anyway, I've opened a new pull request that should be fine now with all the edits/additions.

**Comment by Calinou** [comment]
Superseded by https://github.com/godotengine/godot/pull/49344.

**Comment by pycbouh** [comment]
> > Please squash your commits.
> 
> That deadly advice :D

They've done a couple of force-pushes completely fine! But oh well...

**Pull request closed without merging**